lua/tardis/libraries/sh_messages.lua

```diff
@@ -0,0 +1,28 @@
+CreateConVar("tardis2_debug", 0, {FCVAR_ARCHIVE, FCVAR_REPLICATED}, "TARDIS - debug enabled")
+
+function TARDIS:IsDebugOn()
+	return GetConVar("tardis2_debug"):GetBool()
+end
+
+function TARDIS:Debug(text)
+	if TARDIS:IsDebugOn() then
+		local fulltext = "[TARDIS Debug]"..text
```

Comment:
Should probably be a space here before the message
```suggestion
		local fulltext = "[TARDIS Debug] "..text
```